bandwidth2/bandwidth2.c

```diff
@@ -77,27 +78,36 @@ void get_values(char **const ifaces, int num_ifaces, time_t * const s, ulli * co
   }
 }
 
-void display(int const unit, double b, int const warning, int const critical)
+void display(int const unit, int const divisor,
+             double b, int const warning, int const critical)
 {
+  char fmtstr[7];
+
   if (critical != 0 && b > critical) {
-    printf("<span color='%s'>", RED);
+    printf("<span fallback='true' color='%s'>", RED);
```

Comment:
I'm not familiar with the fallback option for pango, what does this do?

Reply:
Apparently allows fallback to other fonts if the specified font doesn't support a given unicode code point. I think it may have fixed it because I use a font-awesome symbol as a LABEL, and perhaps font-awesome doesn't have a code point for space (0x20)? Not entirely sure.